client/js/components/statement/assessmentTable/FilterModalSelectItem.vue

```diff
@@ -9,7 +9,7 @@
 
 <template>
   <div
-    v-if="false === hidden"
+    v-if="false === hidden && (dplan.projectName === 'Planfeststellungsverfahren' && filterItem.attributes.label !== 'Datei')"
```

Comment:
Afaik we want to avoid using the `projectName` for condition checks like this (`if projectName === 'someString'`) because we don't want to have the project names in the code. I think we should create a permission for this and handle it in the backend

Reply:
This is what I was afraid of. Thanks, I will get in touch with the php team for that and adjust this afterwards with this PR.

Reply:
Now using the permission check and a computed to determine if the file filter should be rendered here: https://github.com/demos-europe/demosplan-core/pull/3751/commits/6e4a4891d083e58cdb37865940f474a0175d75c7
(the permission is only disabled for the planfestsh project)

And removed a duplicate of the computed here https://github.com/demos-europe/demosplan-core/pull/3751/commits/f3119bf98e268f190600e906b6774e699a9f4284 (which i forgot to delete after fixing the sort order :/ )

Reply:
I think that we don't really need to do anything in the FE; if I understand correctly, the permission can be checked in `getAvailableFiltersAssessmentTable` in the BE. Or am I missing something?

Reply:
i agree with @hwiem , i think this can be only in BE handled

Reply:
Thanks, I did not know of the existence of the `getAvailableFiltersAssessmentTable` function but it is the cleaner solution indeed, removed the frontend checks here: https://github.com/demos-europe/demosplan-core/pull/3751/commits/0ec36c5685baa61b21f9b17c7b652679f74227bf and @jalalifardemos will handled the BE part of this PR.